doc/Language/regexes.pod6

```diff
@@ -1255,6 +1255,18 @@ list of predefined subrules is listed in
 L<S05-regex|https://design.perl6.org/S05.html#Predefined_Subrules> of design
 documents.
 
+=head1 X«Regex Interpolation|Regex Interpolation; <$patten>»
+
+If you want to build a regex using pattern given at the runtime, regex
+interpolation is what you looking for.
+
+    sub runtime-regex(Str $patten --> Regex) { /<$patten>/ }
```

Comment:
patten

Reply:
not sure if you want to convert 'regex' into 'patten', plz check it.

Reply:
well, isn't it supposed to be “patteRn”? :)

Reply:
see it, sorry :P